smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -200,159 +222,296 @@ public Task Install()
             agentImage.ForEach(
                 image =>
                 {
-                    Console.WriteLine($"Setting up iotedged with agent image {image}");
+                    Console.WriteLine($"Setting up aziot-edged with agent image {image}");
                 },
                 () =>
                 {
-                    Console.WriteLine("Setting up iotedged with agent image 1.0");
+                    Console.WriteLine("Setting up aziot-edged with agent image 1.0");
                 });
 
-            const string YamlPath = "/etc/iotedge/config.yaml";
-            Task<string> text = File.ReadAllTextAsync(YamlPath);
-            var doc = new YamlDocument(await text);
+            const string KEYD = "/etc/aziot/keyd/config.toml";
+            const string CERTD = "/etc/aziot/certd/config.toml";
+            const string IDENTITYD = "/etc/aziot/identityd/config.toml";
+            const string EDGED = "/etc/aziot/edged/config.yaml";
+
+            Dictionary<string, (string owner, IConfigDocument document)> config = new Dictionary<string, (string, IConfigDocument)>();
+            config.Add(KEYD, ("aziotks", InitDocument(KEYD + ".default", true)));
+            config.Add(CERTD, ("aziotcs", InitDocument(CERTD + ".default", true)));
+            config.Add(IDENTITYD, ("aziotid", InitDocument(IDENTITYD + ".default", true)));
+            config.Add(EDGED, ("iotedge", InitDocument(EDGED + ".template", false)));
```

Comment:
Maybe it is a good idea to add a comment here referring to a readme doc regarding these keys & use cases: `aziotks`, `aziotcs`, `aziotid`, `iotedge`.